persist/fs/manager.go

```diff
@@ -82,21 +82,21 @@ func (pm *persistManager) persist(
 	rateLimitMbps := pm.rateLimitOpts.LimitMbps()
 	if pm.rateLimitOpts.LimitEnabled() && rateLimitMbps > 0.0 {
```

Comment:
Meh these are literally one alone getters so the compiler should already inline these anyway